releasetasks/templates/firefox/bb_update_verify.yml.tmpl

```diff
@@ -10,6 +10,9 @@
         - {{ stableSlugId(upstream_builder) }}
         {% endfor %}
         - {{ stableSlugId("release-{}-{}_updates".format(branch, product)) }}
+        {% if update_verify_requires_cdn_push is defined and update_verify_requires_cdn_push %}
```

Comment:
I tried it and not sure if it looks easier to read... https://gist.github.com/rail/9c135ff75524df54e35901e23e48d5d9

Also the tests would become more complicated... Let's go with the current version!